tests/functional/test_report.py

```diff
@@ -5,160 +5,159 @@ def test_report_allchanges_karmabot(karmabot_dir, capfd):
-        "- Bob Belderbos                        |   3",
-        "- PyBites                              |   1",
-        "- Patrick-Oliver Groß                  |   2",
-        "2019-46",
-        "- Bob Belderbos                        |  14",
-        "- Patrick-Oliver Groß                  |   3",
-        "2019-49",
-        "- Patrick-Oliver Groß                  |   1",
-        "- AJ Kerrigan                          |   1",
-        "2020-00",
-        "- AJ Kerrigan                          |   2",
-        "- Bob Belderbos                        |   1",
-        "2020-17",
-        "- Patrick-Oliver Groß                  |   1",
-        "2020-19",
-        "- Patrick-Oliver Groß                  |   1",
-        "2020-27",
-        "- Bob Belderbos                        |   1",
-        "2020-28",
-        "- Bob Belderbos                        |   1",
-        "- Patrick-Oliver Groß                  |   1",
-        "2020-32",
-        "- Bob Belderbos                        |   3",
-        "- Patrick-Oliver Groß                  |   1",
-        "2020-35",
-        "- Bob Belderbos                        |   1",
-        "2020-36",
-        "- Patrick-Oliver Groß                  |   4",
-        "- Bob Belderbos                        |   5",
-        "- Michael Aydinbas                     |   3",
-        "- AJ Kerrigan                          |   2",
-        "2020-39",
-        "- Patrick-Oliver Groß                  |   2",
-        "- Michael Aydinbas                     |   2",
-        "2020-40",
-        "- Michael Aydinbas                     |   1",
-        "- Ivan Gonzalez                        |   1",
-        "2020-51",
-        "- Bob Belderbos                        |   1",
-        "- Patrick-Oliver Groß                  |   1",
-        "2021-12",
-        "- Patrick-Oliver Groß                  |   1",
-        "2021-22",
-        "- pre-commit-ci[bot]                   |   1",
-        "",
-        "* Files that are most often found in commits:",
-        "",
-        "bot/slack.py                           |  77",
-        "README.md                              |  28",
-        "bot/karma.py                           |  21",
-        "commands/topchannels.py                |  17",
-        "tests/test_bot.py                      |  14",
-        "main.py                                |  14",
-        ".gitignore                             |  13",
-        "pyproject.toml                         |  12",
-        "test/tests.py                          |  12",
-        "requirements.txt                       |  11",
-        "--------------------------------------------------",
-    ]
+     '--------------------------------------------------'
+    , '< Git Repo Activity Report >'
+    , '> Repo: karmabot'
+    , '--------------------------------------------------'
+    , ''
+    , '* Repo changes (inserts and deletes) per week:'
+    , ''
+    , '2017-25 ████                                 | 248'
```

Comment:
This looks really nice, I wonder if we drop the numbers? Another thing we could introduce is more aggregation (month, year) for bigger projects, but we can list that as a new enhancement.

Reply:
I personally like the numbers along with the bars and my vote would be to keep them. Mostly because it's hard to know the true scale with bars alone. However, if you would like me to make this change, I'm more than happy to do so!

Reply:
Yeah makes sense, let's leave the numbers for now, thanks.